packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -565,4 +571,38 @@ describe('Table', () => {
             );
         });
     });
+
+    describe('hidden columns', () => {
+        it('does not render hidden columns', async () => {
+            column1.hidden = true;
+            element.setData(simpleTableData);
+            await connect();
+            await waitForUpdatesAsync();
+
+            verifyRenderedData(simpleTableData);
```

Comment:
I don't think this test is sufficient to verify that the hidden columns aren't rendering, since `verifyRenderedData` skips hidden columns, and there's no additional verification on the number of columns that have rendered.

You could additionally check the number of headers, excluding hidden columns, and make sure that's the right number:
```ts
const visibleColumnCount = element.columns.filter(x => !x.hidden).length;
expect(pageObject.getRenderedHeaderCount()).toEqual(visibleColumnCount);
```
That still assumes that the rendered header count will match the rendered cell count, for a row. I guess we could add something on the page object to get the number of cells for a row (assuming there's at least 1 row to check), if we wanted to verify that the # of rendered cells in a row equals the number of non-hidden columns.

Reply:
That's a good point. I went back and forth with the best way to test this.

I've gone ahead and added a new function to the page object to get the number of cells in a row. I'm now asserting that the cell count and header count is correct within each of the new tests.